src/builder/decision_nnf_builder.rs

```diff
@@ -217,10 +217,10 @@ impl DecisionNNFBuilder {
             }
         } else {
             // check cache
-            let _idx = match bdd.get_scratch::<BddPtr>() {
-                None => (),
-                Some(v) => return if bdd.is_neg() { v.neg() } else { *v },
-            };
```

Comment:
This is the only code removal in the PR. Not clear what this does (`_idx` is never used). Lmk if I should just `#allow` this instead!